test/Dockerfile.test.php8

```diff
@@ -25,10 +18,50 @@ RUN apt-get install -y perl && \
 # Update the package list and install required packages
 
 RUN apt-get update && \
-    apt-get install -y \
-    python3 \
-    python3-dev \
-    python3-pip \
-    libmariadb-dev-compat \
-    libmariadb-dev \
-    virtualenv
+    apt-get install -y mariadb-client libzip-dev
+
+RUN docker-php-ext-install pdo_mysql && \
+    docker-php-ext-install zip
+
+ARG projectname=Loris-MRI
+RUN useradd -ms /bin/bash lorisadmin \
+    && mkdir -p /data/${projectname} \
+    && mkdir -p /opt/${projectname}/bin/mri \
+
+# Update packages and install necessary tools and libraries
+RUN apt-get update \
+    && apt-get install -y \
+        python3 \
+        wget \
+        imagemagick \
+        python3-dev \
+        python3-pip \
+        default-libmysqlclient-dev \
+        virtualenv \
+    && rm -rf /var/lib/apt/lists/*
+
+
+########################
+# Install MINC Toolkit #
+########################
+
+# Install MINC Toolkit dependencies
+RUN apt-get install -y libc6 libstdc++6 imagemagick perl
+
+# Download the MINC Toolkit package
+RUN wget -P /tmp http://packages.bic.mni.mcgill.ca/minc-toolkit/Debian/minc-toolkit-1.9.18-20200813-Debian_10-x86_64.deb
+
+# Install the MINC Toolkit package
+RUN dpkg -i /tmp/minc-toolkit-1.9.18-20200813-Debian_10-x86_64.deb
+
+# Run the MINC Toolkit configuration script 
+RUN . /opt/minc/1.9.18/minc-toolkit-config.sh
```

Comment:
Will we update the config path in a separate PR?

Reply:
Yes, we will do it in a separate PR.